core/store/src/genesis.rs

```diff
@@ -0,0 +1,147 @@
+use rayon::prelude::*;
+use std::{collections::HashSet, fs, path::Path};
+
+use borsh::BorshDeserialize;
+use near_chain_configs::Genesis;
+use near_primitives::{
+    epoch_manager::EpochConfig,
+    runtime::{config::RuntimeConfig, config_store::RuntimeConfigStore},
+    shard_layout::{account_id_to_shard_id, ShardLayout},
+    state_record::{state_record_to_account_id, StateRecord},
+    types::{AccountId, ShardId, StateRoot},
+};
```

Comment:
Note to myself: it is disturbing how much primitives of different kinds we currently need just to initialize genesis. If we are going to make `near-store` independent at some point, it won't be trivial...